.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: Test and lint
```

Comment:
Maybe just a very generic short name

```suggestion
name: CI
```

I don't thin that the workflow name is that important. the jobs names are more important.

Maybe GitHub will consider updating the UI to put the workflow name at the end :)

Reply:
We have a second workflow, so the name should be descriptive to a certain degree.

Reply:
The way I use this in my own projects is to have a name for the workflow file but not for the jobs, as the key name should be descriptive enough. If you want, I can give that a try

Reply:
@adiroiban Shortened the workflow name and removed the job names. Hope that fixes your issue :+1: